code/game/objects/items/inducer.dm

```diff
@@ -247,22 +251,9 @@
 	powerdevice = /obj/item/stock_parts/power_store/battery/super
 
 /obj/item/inducer/cyborg
-	name = "internal inducer"
+	name = "modular inducer"
 	icon = 'icons/obj/tools.dmi'
 	icon_state = "inducer-engi"
-	powerdevice = null
-
-/obj/item/inducer/cyborg/add_context(atom/source, list/context, obj/item/held_item, mob/user)
```

Comment:
Why was this removed.

The tooltips for regular inducer don't apply for borgs so this should return NONE